cluster/internal/data.proto

```diff
@@ -2,7 +2,8 @@ package internal;
 
 message WriteShardRequest {
     required uint64 ShardID = 1;
-    repeated Point Points = 2;
+    required uint64 OwnerID = 2;
```

Comment:
I was trying to get an interface to fit and I could find the node id where I needed it. I agree that this doesn't feel right to me either.